apps/retrospective/services/sprint_task_member.go

```diff
@@ -63,6 +63,34 @@ func (service SprintTaskMemberService) GetMembers(
 	return members, http.StatusOK, nil
 }
 
+// AddSprintActiveMember ...
+func (service SprintTaskMemberService) AddSprintActiveMember(taskList *retroSerializers.SprintTasksSerializer,
```

Comment:
Please use the sprint task service's `List` method and compute the `My Time` like other fields.

Reply:
done